x-pack/plugins/profiling/server/lib/setup/steps/get_fleet_policy_step.ts

```diff
@@ -101,6 +101,35 @@ export function getFleetPolicyStep({
         ...omit(apmPolicy, 'id', 'revision', 'updated_at', 'updated_by'),
         inputs: modifiedPolicyInputs,
       });
+
+      // We add here the creation of the new package_policy for symbolizer.
+      // We create the new policy and bind it to the Cloud default agent policy;
+      // to do so, force s required to be set to true.
+      const cloudAgentPolicyId = 'policy-elastic-agent-on-cloud';
+      await packagePolicyClient.create(
+        soClient,
+        esClient,
+        {
+          policy_id: cloudAgentPolicyId,
+          enabled: true,
+          package: {
+            name: 'profiler_symbolizer',
+            title: 'Universal Profiling Symbolizer',
+            version: '8.8.0-preview',
```

Comment:
you could consider omitting this field altogether, this will make it always install the latest package it finds (including prerelease packages) that is compatible with this version of Kibana

this means that this version field will not need to be manually update for later releases, including patches. but it also means care should be given as to not accidentally release a package incorrectly (i.e. publishing a `8.9.0` without a >8.9.0 kibana version constraint)

Reply:
@jen-huang whan you say `omitting this field` i suppose you mean not setting `package` at all? Or does it mean leaving it as an empty object o `null`?

Reply:
Erased in https://github.com/elastic/kibana/pull/156089/commits/7d70af940407e84423999818de02d8984d1ea795

Reply:
Apparently this commit broke the tests... I'll remove it and re-run them

Reply:
I only meant removing `version: '8.8.0-preview',` line :)

Reply:
Ah thanks, good to know 😄 
Will keep in mind for the next PR 👍🏼